vectors_handling/uvg.py

```diff
@@ -258,3 +261,85 @@ def generate_top_unigrams(save_path):
 
     top_unigrams = heapq.nlargest(1000, all_unigrams, key=all_unigrams.get)  # fetch top 1000 unigrams
     util.save_file(save_path, top_unigrams)
+
+
+def function_words_process(user_dir):
+    user_vector = [0] * 1000
```

Comment:
User vectors are the same length as the number of feature words (426 in our case)